qa/sc_evm_feeHistory.py

```diff
@@ -127,10 +127,17 @@ def run_test(self):
         history_pending_sc_node_1 = sc_node_1.rpc_eth_feeHistory("0x10", "pending", [25.0, 75.0])
         history_pending_sc_node_2 = sc_node_2.rpc_eth_feeHistory("0x10", "pending", [25.0, 75.0])
 
+        history_latest_sc_node_1 = sc_node_1.rpc_eth_feeHistory("0x10", "latest", [25.0, 75.0])
+        history_latest_sc_node_2 = sc_node_2.rpc_eth_feeHistory("0x10", "latest", [25.0, 75.0])
+
         assert_equal(history_pending_sc_node_1['result'], history_pending_sc_node_2['result'], "fee history should be equal among connected nodes")
-        assert_equal(None, history_pending_sc_node_1['result']['baseFeePerGas'])
-        assert_equal(None, history_pending_sc_node_1['result']['gasUsedRatio'])
-        assert_equal(None, history_pending_sc_node_1['result']['reward'])
+        assert_equal(history_latest_sc_node_1['result'], history_latest_sc_node_2['result'], "fee history should be equal among connected nodes")
+        assert_equal(history_pending_sc_node_1['result'], history_latest_sc_node_1['result'], "fee history should be equal for pending and latest nodes")
+
+        assert_equal('0x1', history_pending_sc_node_1['result']['oldestBlock'])
+        assert_equal(9, len(history_pending_sc_node_1['result']['baseFeePerGas']))
```

Comment:
If it is not too much effort, it would be nice to link the expected length of  _gasUsedRatio_, _reward_ and _baseFeePerGas_ result arrays with the height of the blockchain. I mean, the feeHistory requests the latest 16 blocks fee info but it returns just 8. It would be nice to see  where 8 (and 9) number comes from.

Reply:
Done

Reply:
OK, just add a comparison between the height and the number of blocks requested, so the expected result length is the minimum between the 2. E.g.
```
blockCount = 16
history_pending_sc_node_1 = sc_node_1.rpc_eth_feeHistory(str(int(blockCount,16)), "pending", [25.0, 75.0])

....
expected_result_length = blockCount 
if (blockCount> height)
       expected_result_length = height
....
assert_equal(expected_result_length, len(history_pending_sc_node_1['result']['gasUsedRatio']))
```
It's just an example

Reply:
Done